roles/backend_setup/tasks/thin_pool_create.yml

```diff
@@ -1,6 +1,9 @@
 ---
 # Create a logical volume thinpool when thinpoolname is given
-
+- name:
+  package:
+    name: thin-provisioning-tools
+    state: present
```

Comment:
Is this package specific to Debian/Ubuntu? I get error when I run on RHEL/Fedora. Can you please put the check to run only on Debian/Ubuntu?

Reply:
Please move this to main.yml before the vdo code. So that pre-requisites are checked first. Rest looks good.